src/models/tiles/text/text-content.ts

```diff
@@ -22,7 +22,7 @@ export const TextContentModel = TileContentModel
     format: types.maybe(types.string)
   })
   .volatile(self => ({
-    editor:  undefined as Editor | undefined,
+    editor:  undefined as CustomEditor | undefined,
```

Comment:
I'm not sure this is needed. I don't understand the difference between Editor and CustomEditor. I think one might be an alias of the other?

Reply:
`Editor` is defined in the `slate` library. It uses some TypeScript magic to allow the client (in this case our `slate-editor` library) to "extend" the underlying `Editor` interface, so the effect is something like `CustomEditor extends Editor`, but may not be exactly the same.

Reply:
The effect is that using `CustomEditor` here means it'll be typed to have our extensions, which seems like a good thing.